### PR TITLE
PARWDFT [Andrew] [ongoing]

### PR DESCRIPTION
**Pull request by andrewnunezbibby**

### CODE REVIEW

File: starter_code/queries.md [hunk 1]
```diff
@@ -5,79 +5,105 @@
...
 <!-- Your Code Goes Here -->
+filter: {name: "Babelgum"}
+projection: {name: 1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
 <!-- Your Code Goes Here -->
+filter: {number_of_employees: {$gt: 5000}}
+sort: {number_of_employees: 1}
+
```
Comment (ta-web-paris) [inline starter_code/queries.md]: You forgot the limit :p

File: starter_code/queries.md [hunk 2]
```diff
@@ -5,79 +5,105 @@
...
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
+filter: {founded_year: {$gt: 2000}, founded_year: {$lt:2005}}
+projection: {name: 1, founded_year: 1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: Your filter should include the years 2005 and 2000, so it should be $lte and $gte

File: starter_code/queries.md [hunk 3]
```diff
@@ -5,79 +5,105 @@
...
 
 <!-- Your Code Goes Here -->
-
+filter: {category_code: null}
+<!-- RETURNED ALL  RECORDS -->
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
 
 <!-- Your Code Goes Here -->
-
+filter: {number_of_employees: {$gt: 100}, number_of_employees: {$lt:1000}}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: 100 should be included, so it should be $gte 100

File: starter_code/queries.md [hunk 4]
```diff
@@ -5,79 +5,105 @@
...
 
 <!-- Your Code Goes Here -->
-
+sort: {"ipo.valuation_amount": -1}
 ### 10. Retrieve the 10 companies with more employees, order by the `number of employees`
 
 <!-- Your Code Goes Here -->
-
+filter: {number_of_employees: {$gt: 10}}
+sort: {number_of_employees: 1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: You had to retrieve the companies with the most employees, not the companies with more than 10 employees.
To achieve this you could do it by sorting it in a descending way and then limit the search to 10 companies.
sort: {number_of_employees: 1}
limit: 10

File: starter_code/queries.md [hunk 5]
```diff
@@ -5,79 +5,105 @@
...
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
 <!-- Your Code Goes Here -->
+filter: {founded_month: {$gte: 6}}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: you forgot the limit to 1000

File: starter_code/queries.md [hunk 6]
```diff
@@ -5,79 +5,105 @@
...
 
 <!-- Your Code Goes Here -->
+filter: {"acquisition.price_amount": {$gt: 10000000}, "acquisition.price_currency_code":"EUR"}
 
 ### 18. All the companies that have been acquired on the first trimester of the year. Limit the search to 10 companies, and retrieve only their `name` and `acquisition` fields.
 
 <!-- Your Code Goes Here -->
-
+filter: {"acquisition.acquired_month": {$lte: 4}}
+projection:{name: 1, acquisition:1}
```
Comment (ta-web-paris) [inline starter_code/queries.md]: You forgot the limit

**Comment by ta-web-paris** [comment]
Well done, closing the PR

**Comment by andrewnunezbibby** [comment]
@ta-web-paris thanks. I did miss those gte vs gt/ lte vs lt nuances in the prompt language. But I was intentionally omitting limits because i didnt think they needed to be included (because they didnt require coding in Compass, they were just input fields, so I didnt think it needed to be included. My bad!!)

**Pull request closed without merging**